lib/engine.js

```diff
@@ -238,37 +237,50 @@ class Engine extends EventEmitter2 {
         reject(e)
       })
       inkProc.on('exit', () => {
-        this.emit('done.inkscape.export.all', {time: elapsed(inkExecStart)})
+        this.emit('done.inkscape.export.all', { time: elapsed(inkExecStart) })
         resolve()
       })
       // Run all the conversions in the shell
       const inkShellCmd = _.join(commands, ' \n') + ' \nquit\n'
       info('Inkscape shell commands:', inkShellCmd)
-      let inkExecStart = nowInMillis()
+      const inkExecStart = nowInMillis()
       let inkCmdStart = inkExecStart
       inkProc.stdin.write(inkShellCmd)
     })
   }
 
-  _convertWithMuPDF (outputDir, pdfFiles) {
+  /**
+   * @param files An array of PDF or PNG files
```

Comment:
I know this is an old method, but I think it would be helpful to indicate what the method does (i.e. what is being converted into what?)

Reply:
I documented the input and the output, but if that's not clear I'll add another blurb.